idd/Energy+.idd.in

```diff
@@ -67346,7 +67346,7 @@ ThermalStorage:ChilledWater:Stratified,
        \type node
   N13, \field Source Side Heat Transfer Effectiveness
        \type real
-       \minimum 0.0
+       \minimum> 0.0
```

Comment:
Same change needed in ThermalStorage:ChilledWater:Mixed.  And should we be making the same change to minimum> 0.0 for the "Use Side Heat Transfer Effectiveness" in these four objects?